markdown/preface.markdown

```diff
@@ -159,20 +160,24 @@ Although this book will focus on NumPy and SciPy,
 it is the many surrounding packages that make Python a powerhouse for
 scientific computing.
 
-
 ## The Great Cataclysm: Python 2 vs. Python 3
 
 In your Python travels, you may have already heard a few rumblings about which version of Python is better.
 You may have wondered why it's not just the latest version.
 
-At the end of 2008, the Python core developers released Python 3.0, a major update to Python with better Unicode (international) text handling, type consistency, and streaming data handling, among other improvements.
-To quote Douglas Adams in The Hitchhiker's Guide to the Galaxy, "this has made
-a lot of people very angry and been widely regarded as a bad move."
```

Comment:
I reluctantly agree that most people don't get it. On the other hand, @stefanv, the idea you got is correct: Python 3 *was* widely regarded as a flop. It's just that now it's better, and going forward it's only going to get better still.

We could certainly argue to keep it in. We need to convey the idea that a lot of the writing that people will have encountered online about the transition is accurate but out of date. My own student said his impression from reading Python things on the internet was that he should stick to Python 2.7! I was like, errr, not if you're working with me, buddy! =P